packages/dd-trace/src/telemetry/logs/index.js

```diff
@@ -35,18 +35,21 @@ function onLog (log) {
 }
 
 function onErrorLog (msg) {
-  if (msg instanceof Error) {
-    onLog({
-      level: 'ERROR',
-      message: msg.message,
-      stack_trace: msg.stack
-    })
-  } else if (typeof msg === 'string') {
-    onLog({
-      level: 'ERROR',
-      message: msg
-    })
+  const { message, cause } = msg
+  if (!message && !cause) return
+
+  const telLog = {
+    level: 'ERROR',
+
+    // existing log.error(err) without message will be reported as 'Generic Error'
+    message: message || 'Generic Error'
```

Comment:
We are assigning the 'Generic Error' message to all errors logged without message. 
It allows us to not to update all the `log.error(err)` cases in the tracer but will be recommended to provide a message for each of the cases

Reply:
Not sure I understand. What is the current behaviour for this? Are we just logging nothing?

Reply:
It is the message for telemetry logs when there is no message.

We were sending the error message as telemetry log message but guys from telemetry complained about it due to cardinality and possible PII data problems.

Now, with the new log api, it is preferable whenever you log an error that is accompanied by a no dynamic message.
```javascript

// if you call log.error with a message and an error
// logger: 
//  - logs 'Error loading file'
//  - logs e
// telemetry:
//  - sends 'Error loading file' message + e.stack
log.error('Error loading file', e) 

```
But if a message is not provided `'Generic Error'` message is used

```javascript
// logger:
// - logs e
// telemetry:
// - sends 'Generic Error' message + e.stack
log.error(e) 
```

Reply:
Then this logic should be pushed to telemetry and not to the publisher.